python/pyspark/ml/tests.py

```diff
@@ -929,6 +932,50 @@ def test_apply_binary_term_freqs(self):
                                    ": expected " + str(expected[i]) + ", got " + str(features[i]))
 
 
+class ALSTest(PySparkTestCase):
+
+    def test_storage_levels(self):
```

Comment:
Ok will test just param setters. Just for clarity, what do you mean by "set the value on the JVM side"? Do you want to check that `getIntermediateRDDStorageLevel` returns the correct value, or check that the value of the underlying JavaEstimator param is correct?

 By the way, I have made these params part of `ALSParams` and not `ALSModelParams` because setting them in the model does nothing. However, I could provide getters in the model, what do you think?